tables/adsb-uncertainty-parameters.tgn

```diff
@@ -1 +1 @@
[]},{"value":"Paramter","cspan":1,"rspan":1,"markup":[1,8]},{"value":"Acronym","cspan":1,"rspan":1,"markup":[1,7]}],[{"value":"NUCp\n(V0,1,2)","cspan":1,"rspan":3,"markup":[1,13]},{"value":"Horizontal Protection Limit","cspan":1,"rspan":1,"markup":[1,27]},{"value":"HPL","cspan":1,"rspan":1,"markup":[1,3]}],[{"value":"","cspan":1,"rspan":-1,"markup":[]},{"value":"95\\% Containment radius on horizontal position error","cspan":1,"rspan":1,"markup":[1,52]},{"value":"RCxy","cspan":1,"rspan":1,"markup":[1,4]}],[{"value":"","cspan":1,"rspan":-2,"markup":[]},{"value":"95\\% Containment Radius on vertical position error","cspan":1,"rspan":1,"markup":[1,50]},{"value":"RCz","cspan":1,"rspan":1,"markup":[1,3]}],[{"value":"NUCr\n(V0)","cspan":1,"rspan":2,"markup":[1,9]},{"value":"Horizontal Velocity Error (95\\%)","cspan":1,"rspan":1,"markup":[1,32]},{"value":"HVE","cspan":1,"rspan":1,"markup":[1,3]}],[{"value":"","cspan":1,"rspan":-1,"markup":[]},{"value":"Vertical Velocity Error (95\\%)","cspan":1,"rspan":1,"markup":[1,30]},{"value":"VVE","cspan":1,"rspan":1,"markup":[1,3]}],[{"value":"NACp\n(V1,2)","cspan":1,"rspan":2,"markup":[1,11]},{"value":"Estimated Position Uncertainty (95\\% horizontal accuracy - p)\na.k.a. Horizontal Figure of Merit (HFOM) in GNSS","cspan":1,"rspan":1,"markup":[1,110]},{"value":"EPU\n","cspan":1,"rspan":1,"markup":[1,4]}],[{"value":"","cspan":1,"rspan":-1,"markup":[]},{"value":"Vertical Estimated Position Uncertainty (95\\% vertical accuracy - p)\na.k.a. Vertical Figure of Merit (VFOM) in GNSS","cspan":1,"rspan":1,"markup":[1,115]},{"value":"VEPU\n","cspan":1,"rspan":1,"markup":[1,5]}],[{"value":"NACv\n(V1,2)\n","cspan":1,"rspan":2,"markup":[1,12]},{"value":"Horizontal Figure of Merit (95\\% horizontal accuracy - v)","cspan":1,"rspan":1,"markup":[1,57]},{"value":"HFOMr","cspan":1,"rspan":1,"markup":[1,5]}],[{"value":"","cspan":1,"rspan":-1,"markup":[]},{"value":"Vertical Figure of Merit (95\\% horizontal accuracy - v)","cspan":1,"rspan":1,"markup":[1,55]},{"value":"VFOMr","cspan":1,"rspan":1,"markup":[1,5]}],[{"value":"NIC\n(V1,2)\n","cspan":1,"rspan":2,"markup":[1,11]},{"value":"Horizontal containment radius limit","cspan":1,"rspan":1,"markup":[1,35]},{"value":"RC","cspan":1,"rspan":1,"markup":[1,2]}],[{"value":"","cspan":1,"rspan":-1,"markup":[]},{"value":"Vertical Protection Limit\n","cspan":1,"rspan":1,"markup":[1,26]},{"value":"VPL","cspan":1,"rspan":1,"markup":[1,3]}],[{"value":"SIL\n(V1,2)\n","cspan":1,"rspan":2,"markup":[1,11]},{"value":"Probability of exceeding horizontal containment radius","cspan":1,"rspan":1,"markup":[1,54]},{"value":"P-RC","cspan":1,"rspan":1,"markup":[1,4]}],[{"value":"","cspan":1,"rspan":-1,"markup":[]},{"value":"Probability of exceeding vertical integrity containment region","cspan":1,"rspan":1,"markup":[1,62]},{"value":"P-VPL","cspan":1,"rspan":1,"markup":[1,5]}]]},"theme":null,"fixed_layout":false,"markup":{"instances":[{},{"style":{}}]},"options":{"table_caption":"","table_label":"tab:my-table"}}
```

Comment:
Here I only replaced `VHPM` with `VFOM` in the sentence: `Vertical Figure of Merit (VHPM) in GNSS`